src/test/common/utils/numericUtils.ts

```diff
@@ -32,6 +32,22 @@ describe('toNumberOrUndefined', () => {
     it('string given', () => {
       expect(toNumberOrUndefined('lalala')).to.be.eq(undefined)
     })
+
+    it('given 1110,100', () => {
```

Comment:
A `given string 1110,100` would be more consistent. You could also remove string from other places as string should be the default in use. The cases you tests passing numeric value might have 'numeric' added to it to highlight unusual use case.